docs/getting-started/get-started-nuxt-3.md

```diff
@@ -74,94 +68,218 @@ Kendo UI for Vue includes [four artfully designed themes](slug:themesandstyles)
 
-```js
-import { productsData } from '../appdata/products';
-import { process, DataResult, State, CompositeFilterDescriptor, SortDescriptor } from '@progress/kendo-data-query';
-import { Grid as grid } from '@progress/kendo-vue-grid';
-```
+    ```js
+    <script>
+        import { productsData } from '../appdata/products';
+        import { process, type DataResult, type State, type SortDescriptor } from '@progress/kendo-data-query';
+        import { Grid as grid } from '@progress/kendo-vue-grid';
+    </script>
+    ```
 
 1. Add a `<template>` block with a simple heading and create a Data Grid. Bind it to the `products` data:
 
-```html
-<grid
-  :data-items="products"
-  :columns="columns"
-></grid>
-```
+    ```html
+    <template>
+        <h1>Hello Kendo UI for Vue with Nuxt 3!</h1>
+        <grid
+          :data-items="products"
+          :columns="columns"
+        ></grid>
+    </template>
+    ```
 
-1. Add the Data Grid's data and columns in the `setup` section of the `KendoGrid.vue` file:
+1. In the `<script>` section of the `pages\KendoGrid.vue` file:
 
-```js
-const products = productsData;
-const columns = [
-  { field: 'ProductName', title: 'Product Name' },
-  { field: 'UnitPrice', title: 'Price' },
-  { field: 'UnitsInStock', title: 'Units in Stock' },
-  { field: 'Discontinued' }
-] as GridColumnProps[];
-```
+   * Load the `products` file.
+   * Define user friendly column names.
+
+    ```js
+    const products = productsData;
+    const columns = [
+      { field: 'ProductName', title: 'Product Name' },
+      { field: 'UnitPrice', title: 'Price' },
+      { field: 'UnitsInStock', title: 'Units in Stock' },
+      { field: 'Discontinued' }
+    ] as GridColumnProps[];
+    ```
+
+After completing all the steps above, your `KendoGrid.vue` will look like this:
+
+    ```js
+    import { productsData } from '../appdata/products';
+    import { process, type DataResult, type State, type SortDescriptor } from '@progress/kendo-data-query';
+    import { Grid as grid, type GridColumnProps, type GridDataStateChangeEvent } from '@progress/kendo-vue-grid';
+
+    <script>
+    const products = productsData;
+    const columns = [
+      { field: 'ProductName', title: 'Product Name' },
+      { field: 'UnitPrice', title: 'Price' },
+      { field: 'UnitsInStock', title: 'Units in Stock' },
+      { field: 'Discontinued' }
+    ] as GridColumnProps[];
+    </script>
+    
+    <template>
+     <h1>Hello Kendo UI for Vue with Nuxt 3!</h1>
+     <grid :data-items="products" :columns="columns"></grid>
+    </template>
+    ```
 
-These steps let you render a very basic Grid by running `npm run dev` and navigating to the local URL displayed in the terminal.
+This sample code lets you run an application with a very basic Grid:
+
+1. Execute the `npm run dev` command.
+1. Navigate to the local URL displayed in the terminal.
```

Comment:
The app doesn't run with this code, multiple errors are thrown about `KendoGrid.vue`.